.babelrc.server.json

```diff
@@ -5,7 +5,7 @@
       "@babel/preset-env",
       {
         "targets": {
-          "node": 10
+          "node": 12
```

Comment:
I checked that all of our services are at least node 12... maybe we just go straight to 14 as it's the latest LTS?